src/common/schemas/CheckDiscriminator.yml

```
@@ -5,6 +5,6 @@
   discriminator:
     propertyName: type
     mapping:
-      deadman: "./DeadmanCheck.yml"
-      threshold: "./ThresholdCheck.yml"
-      custom: "./CustomCheck.yml"
+      deadman: "#/components/schemas/DeadmanCheck"
```

These look like bad references that likely won't resolve. Should they be something like the following (not sure if they'll actually work, but then again they currently don't..):

```suggestion
      deadman: 
        $ref: "./DeadmanCheck.yml"
```

---

it might actually just be:
```suggestion
      $ref: "./DeadmanCheck.yml"
```

---

Following suggestion produces invalid OpenAPI schema:

```suggestion
       - $ref: "./DeadmanCheck.yml"
```

`oss.yml`:

```yaml
CheckDiscriminator:
      oneOf:
        - $ref: '#/components/schemas/DeadmanCheck'
        - $ref: '#/components/schemas/ThresholdCheck'
        - $ref: '#/components/schemas/CustomCheck'
      discriminator:
        propertyName: type
        mapping:
          - $ref: '#/components/schemas/DeadmanCheck'
          - $ref: '#/components/schemas/ThresholdCheck'
          - $ref: '#/components/schemas/CustomCheck'
```

- https://swagger.io/specification/#discriminator-object

What do you think about?

```suggestion
      deadman: DeadmanCheck
```

```yaml
CheckDiscriminator:
      oneOf:
        - $ref: '#/components/schemas/DeadmanCheck'
        - $ref: '#/components/schemas/ThresholdCheck'
        - $ref: '#/components/schemas/CustomCheck'
      discriminator:
        propertyName: type
        mapping:
          deadman: DeadmanCheck
          threshold: ThresholdCheck
          custom: CustomCheck
```